components/graphs/ThresholdGraph.tsx

```diff
@@ -215,11 +218,22 @@ const ThresholdGraph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
                 axisLine={false}
                 tick={false}
               />
+              <Tooltip
+                wrapperStyle={{ outline: 'none' }}
+                contentStyle={{ backgroundColor: '#2B2B2B90', opacity: 1 }}
+                labelStyle={{ color: 'white' }}
+                formatter={(
+                  value: ForecastDataPoint['expected_generation_kw']
+                ) => [parseFloat(value.toFixed(5)), 'kW']}
+                labelFormatter={(
+                  point: ForecastDataPoint['target_datetime_utc']
+                ) => weekdayFormatter.format(new Date(point))}
```

Comment:
This is always giving "Wed 6:00 PM" for me 
<img width="713" alt="Screen Shot 2023-03-30 at 10 10 44 PM" src="https://user-images.githubusercontent.com/62641231/229013242-49541bb1-28ae-4bd4-b080-f1b6211d721b.png">
<img width="682" alt="Screen Shot 2023-03-30 at 10 10 52 PM" src="https://user-images.githubusercontent.com/62641231/229013267-89d77236-0060-42ae-a9fd-fd374b41c2aa.png">